app/src/main/res/values/integers.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <integer name="dialog_appear_animation_duration">300</integer>
+    <integer name="dialog_disappear_animation_duration">200</integer>
```

Comment:
They different because if dialog appears in 200 ms it's looks laggy and if disappears in 300 it looks slow